services/drupal/web/modules/custom/epa_workflow/epa_workflow.module

```diff
@@ -229,6 +230,28 @@ function epa_workflow_form_alter(array &$form, FormStateInterface $form_state, $
   }
 }
 
+/**
+ * Implements hook_form_BASE_FORM_ID_alter().
+ */
+function epa_workflow_form_node_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $node = $form_state->getFormObject()->getEntity();
+  if ($node instanceof NodeInterface && !$node->isNew()) {
+
+    // Loading will always return the "Current revision".
+    /** @var \Drupal\node\NodeInterface $current */
+    $current = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->load($node->id());
+
+    // The revision ID of the "Current revision" will be lower (older) than the most recent revision.
+    if ($node->getRevisionId() > $current->getRevisionId()) {
+      $url = $node->toUrl('version-history')->toString();
+      $markup = Markup::create("<strong>Note:</strong> The revision you are editing is not the currently published content. It may include edits that are not visible on the published page. Refer to the <a target='_blank' href='$url'>revision page</a> to see other revisions and revert if necessary.");
```

Comment:
I'm thinking we should use t() here rather than the Markup class.

Reply:
Removed usage of Markup and am now using the translation `t()` function as suggested.